lib/catch.hpp

```diff
@@ -13,6 +13,7 @@
 // start catch.hpp
 
 
+#include <stdrandom.h>
```

Comment:
I think that we do not need those changes anymore:
* we should not be depending on any random functionality in catch
* I would prefer if we can avoid making changes to Catch as it makes picking up changes potentially harder. Also note that we have other dependencies on `<random>` (grep -r to see where those are, I am seeing libmedida and tracy potentially)

Reply:
yeah, I have a PR open on medida actually waiting on your review already: https://github.com/stellar/medida/pull/23

I figured we could ignore tracy since it's not something we read back anywhere in our code and we frequently compile it out. if you want I can patch it too though.

I'll remove the catch change and install a `make check` step that greps code for use of these things in the future.